Laravel.gitignore

```diff
@@ -1,3 +1,15 @@
+#Exclude vendor libraries.
+#Including composer.lock and install using composer install.
+#Comment if you want to include all libraries
+/vendor
+
+#Node related
+/node_modules
+
+#Homestead related
+Homestead.yaml
```

Comment:
Since we commit composer.loc we dont need vendor directory. It will be created via a composer install.

homestead.yaml its created when you npm install , but i havent verified this myself.